helix-rest/src/main/java/org/apache/helix/rest/client/CustomRestClientImpl.java

```diff
@@ -122,18 +144,23 @@ protected JsonNode getJsonObject(HttpResponse httpResponse) throws IOException {
 
   @VisibleForTesting
   protected HttpResponse post(String url, Map<String, String> payloads) throws IOException {
+    HttpPost postRequest = new HttpPost(url);
```

Comment:
Is it safe to move the request part out of try catch?

Reply:
Yes, in general, it's good to keep try catch block concise.